bt_navigator SIGSEGV when launching demo-world

OS: Linux Ubuntu 24.04 

Following reported after running `docker compose up demo-world`

```
demo-world-1  | [component_container_isolated-7] [INFO] [1738969871.575383001] [bt_navigator]: Activating
demo-world-1  | [component_container_isolated-7] Magick: abort due to signal 11 (SIGSEGV) "Segmentation Fault"...
demo-world-1  | [ERROR] [component_container_isolated-7]: process has died [pid 102, exit code -6, cmd '/opt/ros/jazzy/lib/rclcpp_components/component_container_isolated --ros-args --log-level info --ros-args -r __node:=nav2_container --params-file /tmp/launch_params_nfm7amzk --params-file /tmp/launch_params_n_i7ai_3 -r /tf:=tf -r /tf_static:=tf_static'].
demo-world-1  | [rviz2-6] [INFO] [1738969873.320085226] [rviz]: Message Filter dropping message: frame 'base_scan' at time 0.006 for reason 'discarding message because the queue is full'

```

@sea-bass , just refreshed Jazzy on my dev box. I'm still seeing issue with `nav2_container`. 

```
demo-world-1  | [ERROR] [component_container_isolated-7]: process has died [pid 102, exit code -6, cmd '/opt/ros/jazzy/lib/rclcpp_components/component_container_isolated --ros-args --log-level info --ros-args -r __node:=nav2_container --params-file /tmp/launch_params_ghfgg7ba --params-file /tmp/launch_params_zo6t5yal -r /tf:=tf -r /tf_static:=tf_static'].
```
Full log attached. If it is TF related, I'm wondering whether these warnings need to be looked at. 

```
demo-world-1  | [component_container_isolated-7] [INFO] [1739206370.819546078] [local_costmap.local_costmap]: Timed out waiting for transform from base_link to odom to become available, tf error: Invalid frame ID "odom" passed to canTransform argument target_frame - frame does not exist
demo-world-1  | [component_container_isolated-7] [INFO] [1739206371.319545534] [local_costmap.local_costmap]: Timed out waiting for transform from base_link to odom to become available, tf error: Invalid frame ID "odom" passed to canTransform argument target_frame - frame does not exist
demo-world-1  | [gz-4] Warning [Utils.cc:132] [/sdf/model[@name="turtlebot3_waffle"]/link[@name="imu_link"]/sensor[@name="tb3_imu"]/gz_frame_id:<data-string>:L51]: XML Element[gz_frame_id], child of element[sensor], not defined in SDF. Copying[gz_frame_id] as children of [sensor].
demo-world-1  | [gz-4] Warning [Utils.cc:132] [/sdf/model[@name="turtlebot3_waffle"]/link[@name="base_scan"]/sensor[@name="hls_lfcd_lds"]/gz_frame_id:<data-string>:L138]: XML Element[gz_frame_id], child of element[sensor], not defined in SDF. Copying[gz_frame_id] as children of [sensor].
demo-world-1  | [gz-4] Warning [Utils.cc:132] [/sdf/model[@name="turtlebot3_waffle"]/link[@name="camera_link"]/sensor[@name="intel_realsense_r200_depth"]/gz_frame_id:<data-string>:L368]: XML Element[gz_frame_id], child of element[sensor], not defined in SDF. Copying[gz_frame_id] as children of [sensor].
demo-world-1  | [component_container_isolated-7] [INFO] [1739206371.819544359] [local_costmap.local_costmap]: Timed out waiting for transform from base_link to odom to become available, tf error: Invalid frame ID "odom" passed to canTransform argument target_frame - frame does not exist

```

[nav2_container_crash.log](https://github.com/user-attachments/files/18737790/nav2_container_crash.log)

--- a/dependencies.repos
+++ b/dependencies.repos
@@ -4,25 +4,3 @@ repositories:
     type: git
     url: https://github.com/ros-navigation/nav2_minimal_turtlebot_simulation.git
     version: main
-
-  # py_trees and related packages for Python based behavior trees.
-  py_trees:
-    type: git
-    url: https://github.com/splintered-reality/py_trees.git
-    version: devel
-  py_trees_js:
-    type: git
-    url: https://github.com/splintered-reality/py_trees_js.git
-    version: devel
-  py_trees_ros:
-    type: git
-    url: https://github.com/splintered-reality/py_trees_ros.git
-    version: devel
-  py_trees_ros_interfaces:
-    type: git
-    url: https://github.com/splintered-reality/py_trees_ros_interfaces.git
-    version: devel
-  py_trees_ros_viewer:
-    type: git
-    url: https://github.com/splintered-reality/py_trees_ros_viewer.git
-    version: devel

--- a/dependencies.repos
+++ b/dependencies.repos
@@ -28,9 +28,3 @@ repositories:
     type: git
     url: https://github.com/sea-bass/py_trees_ros_viewer.git
     version: devel
-
-  # BehaviorTree.CPP for C++ based behavior trees.
-  behaviortree_cpp:
-    type: git
-    url: https://github.com/BehaviorTree/BehaviorTree.CPP.git
-    version: master

--- a/dependencies.repos
+++ b/dependencies.repos
@@ -22,9 +22,7 @@ repositories:
     type: git
     url: https://github.com/splintered-reality/py_trees_ros_interfaces.git
     version: devel
-  # NOTE: For now, using a personal fork to remove deprecation warnings:
-  # https://github.com/splintered-reality/py_trees_ros_viewer/pull/34
   py_trees_ros_viewer:
     type: git
-    url: https://github.com/sea-bass/py_trees_ros_viewer.git
+    url: https://github.com/splintered-reality/py_trees_ros_viewer.git
     version: devel

--- a/tb_autonomy/package.xml
+++ b/tb_autonomy/package.xml
@@ -16,6 +16,8 @@
   <depend>ament_index_cpp</depend>
   <depend>behaviortree_cpp</depend>
   <depend>nav2_msgs</depend>
+  <depend>py_trees_ros</depend>
+  <depend>py_trees_ros_viewer</depend>
   <depend>sensor_msgs</depend>
   <depend>tb_worlds</depend>
   <depend>tf2</depend>

